lib/omnibus/packagers/ips.rb

```diff
@@ -211,6 +239,15 @@ def write_pkg_metadata
         }
       )
 
+      # Append the contents of symlinks.erb if it exists
+      if File.exists?(resource_path("symlinks.erb"))
+        write_symlinks_file
+        add_symlinks = File.read(symlinks_file)
```

Comment:
It would be more efficient to not write to a file and turn around and read from it. Perhaps we can add a new method to the [`Omnibus::Templating` mixin](https://github.com/chef/omnibus/blob/master/lib/omnibus/templating.rb) that can just render some ERB to a string? You could pull the non-file writing functionality out of `Omnibus::Templating::render_template` into a separate method (in the same module) that just returns the rendered content as a string and update `render_template` to use the new method.